server/repository/src/db_diesel/item_row.rs

```diff
@@ -1,6 +1,6 @@
 use super::{item_row::item::dsl::*, unit_row::unit, StorageConnection};
 
-use crate::repository_error::RepositoryError;
+use crate::{item_link, repository_error::RepositoryError};
```

Comment:
In other files I've observed this import being stated as `item_link_row::item_link`, but this is what the rust-analyser suggested fix/syntax was so I just went with it 🙂. Is there a convention I'm missing?

Reply:
`item_link` doesn't have a another module could `item_link`, so it works... but for places that do, then it won't be importing the table...